Mlem/Packages/MlemMiddleware/Sources/MlemMiddleware/Snapshot/Community/Community1Snapshot.swift

```diff
@@ -27,6 +27,10 @@ public struct Community1Snapshot: CacheIdentifiable {
     public let banner: URL?
     public let hidden: Bool
     public let onlyModeratorsCanPost: Bool
+    
+    // This is a dodgy workaround for https://codeberg.org/rimu/pyfedi/issues/882
+    // If that issue gets fixed, we can remove this
```

Comment:
*nit*

So we can find this easily when checking for open TODOs

```suggestion
    // This is a dodgy workaround for https://codeberg.org/rimu/pyfedi/issues/882
    // TODO: If that issue gets fixed, we can remove this
```